tasks/connectors/snyk_v2/snyk_v2.rb

```diff
@@ -124,25 +129,27 @@ def run(opts)
         types = ["vuln"]
         types << "license" if include_license
 
-        issue_filter_json = "{
+        morepages = true
+        issue_json = []
+
+        while morepages
+          pagenum += 1
+          project_ids.each_slice(999) do |sliced_ids|
+            issue_filter_json = "{
```

Comment:
I think project_ids.each_slice(999) is supposed to be the outer loop, and while morepages should be the inner one